ns/init_linux.go

```diff
@@ -68,19 +66,19 @@ func NlHandle() *netlink.Handle {
 
 func getSupportedNlFamilies() []int {
 	fams := []int{syscall.NETLINK_ROUTE}
-	if err := loadXfrmModules(); err != nil {
-		log.Warnf("Could not load necessary modules for IPSEC rules: %v", err)
+	if err := checkXfrmSocket(); err != nil {
+		log.Warnf("Required socket to program IPSEC rules cannot be created: %v. (Requires xfrm_user and xfrm_ago modules)", err)
```

Comment:
Uhm.. You still need to try loading xfrm modules in case the `checkXfrmSocket` fails. This is for cases where the module doesn't get autoloaded.